install/kubernetes/cilium/values.yaml

```diff
@@ -1382,6 +1382,24 @@ monitor:
   # endpoints filtering
   # serviceTopology: false
 
+  # -- LoadBalancer with envoy proxy
+  envoy:
```

Comment:
Rather than calling this section `envoy`, what about something like `layer-7`?

```yaml
loadBalancer:
    layer7:
        backend: envoy | disabled | my-cool-new-proxy
```

Reply:
good point, I just check with other places, we are using `l7` term. For another level of backend, I think it will be a long road as we don't have a good integration right now to inject actual endpoints except envoy.